pkg/controller/framework/informers/factory.go

```diff
@@ -46,15 +46,20 @@ type sharedInformerFactory struct {
 	client        clientset.Interface
 	lock          sync.Mutex
 	defaultResync time.Duration
-	informers     map[reflect.Type]framework.SharedIndexInformer
+
+	informers map[reflect.Type]framework.SharedIndexInformer
+	// startedInformers is used for tracking which informers have been started
+	// this allows calling of Start method multiple times
+	startedInformers map[reflect.Type]bool
```

Comment:
> do we need to listen on the stopCh and clear the map when invoked?

Realistically, there's no coming back from a stop yet.  I'm not even sure what it would mean for things like the indexer and lister references.